examples/lock-app/nrfconnect/main/AppTask.cpp

```diff
@@ -372,6 +379,8 @@ void AppTask::ActionCompleted(BoltLockManager::Action_t aAction)
         LOG_INF("Unlock Action has been completed");
         sLockLED.Set(false);
     }
+
+    sAppTask.UpdateClusterState();
```

Comment:
remove sAppTask here too, sorry I didn't mention previously ;)

Reply:
Are you sure? It will not compile without passing object, as the caller is static method and the called method is not ...

Reply:
Ah, you're right. Haven't noticed it's a static method.